kong/plugins/aws-lambda/handler.lua

```diff
@@ -9,6 +9,8 @@ local meta = require "kong.meta"
 local http = require "resty.http"
 local cjson = require "cjson.safe"
 local public_utils = require "kong.tools.public"
+local aws_lambda_proxy_schema = require "kong.plugins.aws-lambda.proxy_response_schema"
+local schemas = require "kong.dao.schemas_validation"
```

Comment:
Please do not use this module, as it is deprecated in favour of the new schema module written for the new DAO.

Overall, I question the dependency on either schema modules to perform such validations in this plugin (or any other plugin). The schemas (and their validation functions) are not meant for public consumption since there is no guarantee that their API or behaviour won't break in the future. They are meant to declaratively specify DAO entities only, and I fear that using them in plugins breaks the single responsibility principle.

Reply:
Okay I'll change that up. It just seemed like the easiest way to do some validation.

Have any recommendations for how to validate the response format? I can roll my own function with some type() checks and if statements if that works.